packages/auth/src/authcredential.js

```diff
@@ -635,23 +635,31 @@ fireauth.OAuthProvider.prototype.getScopes = function() {
  * Initializes an OAuth AuthCredential. At least one of ID token or access token
  * must be defined. When providing an OIDC ID token with a nonce encoded, the
  * raw nonce must also be provided.
- * @param {?string=} opt_idToken The optional OAuth ID token.
+ * @param {?Object|string} optionsOrIdToken Either the options object containing
+ *     the ID token, access token and raw nonce or the ID token string.
  * @param {?string=} opt_accessToken The optional OAuth access token.
- * @param {?string=} opt_rawNonce The optional raw nonce.
  * @return {!fireauth.AuthCredential} The Auth credential object.
  */
-fireauth.OAuthProvider.prototype.credential = function(opt_idToken,
-    opt_accessToken, opt_rawNonce) {
-  if (!opt_idToken && !opt_accessToken) {
+fireauth.OAuthProvider.prototype.credential =
+    function(optionsOrIdToken, opt_accessToken) {
+  var oauthResponse;
```

Comment:
Nit: Can you please run prettifier through the code? The formatting is off.

Reply:
I believe the prettifier will be automatically triggered before pushing to canonical repo. As you can see, there are 2 commits from the prettier in this PR: 
[AUTOMATED]: Prettier Code Styling
 Do I need to manually run the prettier?